js/script.js

```diff
@@ -35,7 +35,7 @@ var moviesElements = movies.map(function(movies) {
    return React.createElement('li', {key: movies.id},
```

Comment:
change name of argument in callback function ;) in map this will be one movie, not all movies ;)